packages/SwingSet/tools/install-ses-debug.js

```diff
@@ -9,7 +9,7 @@
 
 /* global globalThis */
 
-import 'ses';
+import { lockdown } from '@agoric/install-ses/get-lockdown.js';
```

Comment:
Consider instead `/// <reference types="ses"/>` instead of a getter module.

Reply:
`@agoric/lockdown`